networking/advertise-service-ips.md

```diff
@@ -49,10 +50,23 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 
 ### Before you begin...
 
-- [Configure BGP peering]({{ site.baseurl }}/networking/bgp) between {{site.prodname}} and your network infrastructure
+**Required**
+- [Configure BGP peering]({{site.baseurl}}/networking/bgp) between {{site.prodname}} and your network infrastructure
 - For ECMP load balancing to services, the upstream routers must be configured to use BGP multipath.
 - You need at least one external node outside the cluster that acts as a router, route reflector, or ToR that is peered with calico nodes inside the cluster.
-- Services must be configured with the correct service type (“Cluster” or “Local”) for your implementation. For `externalTrafficPolicy: Local`, the service must be type `LoadBalancer` or `NodePort`.
+- Services must be configured with the correct service mode (“Cluster” or “Local”) for your implementation. For `externalTrafficPolicy: Local`, the service must be type `LoadBalancer` or `NodePort`.
+
+**Limitations**
+- OpenShift, versions 4.5 and 4.6  
```

Comment:
This isn't limited to OpenShift - this is a vanilla kubernetes issue as well. The workaround listed below is enabled by default in OpenShift v4.7+, and can be set in Kubernetes clusters >= v1.18

> where the source IP is not preserved by NodePort service with externalTrafficPolicy:Local or by ClusterIP service, and when using a LoadBalancer service.

I think this isn't correct. The source IP is not preserved by NodePort services with externalTrafficPolicy: Local, but it IS preserved for LoadBalancer services. 

ClusterIP services do not have the source address preserved, but that is working-as-designed according to Kubernetes.

Reply:
Fixed.